info.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "connectorHealthMonitoring",
+    "version": "1.0.0",
+    "type": "solutionpack",
+    "local": true,
+    "label": "Connector Health Monitoring",
+    "dependencies": [],
+    "prerequisite": null,
+    "publisher": "Fortinet",
+    "description": "Solution Pack send the connector health status notification via email based on provided connector list.",
+    "help": null,
+    "category": [],
+    "supportInfo": null,
+    "iconLarge": null,
+    "fsrMinCompatibility": "7.2.0",
+    "date": "2022-08-23T12:29:30+00:00",
+    "contents": {
+        "playbooks": [
+            {
+                "name": "10 - SP - \tConnector Health Monitoring"
```

Comment:
Please make a correction in the PB collection name to remove `\t`

Reply:
Done